sdk/endpoints/objects/v2/query.go

```diff
@@ -0,0 +1,61 @@
+package v2
```

Comment:
question: What is the reason of having a copy of this struct from v1 in v2? couldn't be extracted from v1?

Reply:
exported to `internal/endpoints`